### ISSUE TITLE
Performance vs. JSONKit

### ISSUE THREAD

**Issue by confile** [goodow/GDJson#1]
Did you measure the performance of GDJson vs. https://github.com/johnezang/JSONKit ?

If not, do you have any performance results at all?

**Comment by larrytin** [comment]
No, but GDJson is a very light wrapper for [NSJSONSerialization](http://developer.apple.com/library/ios/#documentation/Foundation/Reference/NSJSONSerialization_Class/Reference/Reference.html).

**Comment by confile** [comment]
According to this site: https://github.com/johnezang/JSONKit

`
Some quick "back of the envelope" calculations using the numbers reported, JSONKit appears to be approximately 25% to 40% faster than NSJSONSerialization, which is pretty significant.
`

**Comment by confile** [comment]
25% is a lot.

**Comment by confile** [comment]
Is there any way to wrap JSONKit instead of NSJSONSerialization?

**Comment by larrytin** [comment]
Sure, the start point is [Json.m](https://github.com/goodow/GDJson/blob/master/Classes/override/com/goodow/realtime/json/Json.m), and then [method-mappings.properties](https://github.com/goodow/realtime/blob/master/resources/j2objc/method-mappings.properties).

**Comment by confile** [comment]
Do you have any plans for that or is the performance good enough for you?

**Comment by larrytin** [comment]
There is no plan to support JSONKit.
